static/app/types/debugFiles.tsx

```diff
@@ -54,4 +52,9 @@ export type AppStoreConnectValidationData = {
    */
   latestBuildVersion: string | null;
   updateAlertMessage?: string;
+  /**
+   * Whether the UI should show an alert indicating we need the user to refresh their iTunes
+   * credentials.
```

Comment:
```suggestion
   * Whether the UI should show an alert indicating we need the user to refresh their iTunes
   * session.
```

isn't it session?

Reply:
yes, the original impl in the backend was overloading credentials everywhere and things are a bit sloppily named sometimes.  thanks for improving 😄